src/VisualStudio/Core/Def/Implementation/FindReferences/AbstractCustomColumnDefinition.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.VisualStudio.Shell.TableControl;
+
+namespace Microsoft.VisualStudio.LanguageServices.CustomColumn
+{
+    /// <summary>
+    /// Implementation of a custom, dynamic column for the Find All References window.
```

Comment:
nit: This isn't really an implementation. Also, what separates a custom column from another column?